src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java

```diff
@@ -0,0 +1,92 @@
+package com.canis.step_definitions;
+
+import com.canis.pages.LoginPage;
+import com.canis.utilities.ConfigurationReader;
+import com.canis.utilities.Driver;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+public class B28G10_87_Login_StepDefinitions {
+
+    LoginPage loginPage = new LoginPage();
+
+    @Given("user is on the login page")
+    public void user_is_on_the_login_page() {
+        Driver.getDriver().get(ConfigurationReader.getProperty("url"));
+    }
+
+    @When("user login as {string}")
+    public void user_login_as(String userType) {
+
+        String username = "marketing1@cybertekschool.com";
```

Comment:
What for we need line 23 and 24?? We using configuration.properties for credentials

Reply:
Ops that is a mistake. That was for the test. I will leave  username = ""    password=""